sass/_header.scss

```diff
@@ -49,12 +49,12 @@
         @media screen and (max-width: 1023px) {
             position: fixed;
             z-index: 800;
-            top: 60px;
+            top: 5.937rem;
```

Comment:
This is a pretty weird value to use ^^
Maybe put it in a variable?

Reply:
The point of using `rem`s was to allow the header to scale up when you zoom in. It's to avoid having the text overflowing from the navbar. I agree a variable would make sense, but I'm not entirely sure if there are best practices for using variables in SCSS. I'm wondering whether to start fiddling with variables right away or to wait for the great post-launch refactoring.

Reply:
Doesn't hurt starting now I would say. After this PR ;)